src/Health/HealthController.ts

```diff
@@ -96,15 +69,7 @@ export class HealthController {
   }
 
   private async updateBlockchainInfo(blockchainInfo: BlockchainInfo): Promise<BlockchainInfo> {
-    await this.collection.updateOne(
-      { name: 'blockchainInfo' },
-      {
-        $set: {
-          blockchainInfo,
-        },
-      },
-      { upsert: true }
-    )
+    await this.healthDAO.updateBlockchainInfo(blockchainInfo)
     return blockchainInfo
```

Comment:
the type of return is wrong...

Reply:
@wzalazar can you help with what's wrong?